.scripts/nightly.yml

```diff
@@ -1,29 +1,30 @@
+trigger: none
 schedules:
   - cron: "0 0 * * *"
```

Comment:
maybe 0 0 is not a very good time to start the task.

Reply:
This would be triggered at midnight in a daily schedule.